exercises/nucleotide-count/.meta/src/reference/java/NucleotideCounter.java

```diff
@@ -27,6 +27,8 @@ private static boolean isCountable(char base) {
   Map<Character, Integer> nucleotideCounts() {
     Map<Character, Integer> counts = emptyCounts();
     for (char c : sequence.toCharArray()) {
+      if (isCountable(c))
```

Comment:
I think it's a bit confusing to have this check here. I would expect it to check that the input is valid in the constructor, so to loop though the sequence in the constructor and throw an exception if any of the characters aren't valid. What do you think?

Also please put curly brackets around the check:
```
if (isCountable(c)) {
  throw new IllegalArgumentException(c + " is not a nucleotide");
}
```
Otherwise it's easy to make mistakes, see https://stackoverflow.com/questions/8020228/is-it-ok-if-i-omit-curly-braces-in-java 🙂 

There's also an instance of this `if` on line twelve without curly braces. It would be great if you wouldn't mind changing that too, though of course you don't have to since you didn't add it 🙂

Reply:
Agree. Working on it